src/helpers/fallbackauthhelper.ts

```diff
@@ -25,11 +28,15 @@ async function processMessage(arg) {
     // We now have a valid access token.
     loginDialog.close();
     const response = await sso.makeGraphApiCall(messageFromDialog.result);
-    writeDataToOfficeDocument(response);
+    callbackFunction(response);
+  } else if (messageFromDialog.error === undefined && messageFromDialog.result.errorCode === undefined) {
+    // Need to pick the user to use to auth
```

Comment:
Should we output some message in that case?

Reply:
No . . . this fixes the problem were the dialog flashes with nothing happening.  The flash happens because the dialog closes prematurely w/out waiting for the consent.  This is block just prevents the dialog from closing while the consent is loading up . . . this is nothing to output as it will come back and fall into one of the other conditional cases.